alpha/alpha-server/src/main/resources/application.conf

```diff
@@ -24,31 +24,37 @@ kamon {
   metric {
     filters {
       trace.includes = ["**"]
+      trace.excludes = []
       trace-segment.includes = ["**"]
       trace-segment.excludes = []
-      akka-actor.includes = []
-      akka-actor.excludes = ["**"]
-      akka-dispatcher.includes = []
-      akka-dispatcher.excludes = ["**"]
+      akka-actor.includes = ["**"]
+      akka-actor.excludes = []
+      akka-dispatcher.includes = ["**"]
+      akka-dispatcher.excludes = []
     }
   }
 
   show-aspectj-missing-warning = no
 
   modules {
     kamon-annotation {
-      auto-start = no
+      auto-start = yes
```

Comment:
Why did you change these settings?

Reply:
while running the alpha server I have to provide auto-start yes for kamon.  I put here yes and forgot to revert. i will make by default "no" and when user required then they will pass argument "yes".